v0x04/utils.py

```diff
@@ -125,6 +125,9 @@ def handle_port_desc(controller, switch, port_list):
                                   speed=port.curr_speed.value,
                                   config=config)
         switch.update_interface(interface)
+        interface_event = KytosEvent(name='kytos/of_core.switch'
+                                     '.interface.created',
+                                     content={'interface': interface})
```

Comment:
It's better to maintain the event name together, so that we can grep the code when needed.

Reply:
Resolved in https://github.com/kytos/of_core/pull/131/commits/9597cd4ef7eae4ef566d8a82be5496e6a71b8268